### ISSUE TITLE
Missing underscore dependency (?)

### ISSUE THREAD

**Issue by csillag** [tmeasday/check-npm-versions#18]
After updating to Meteor version 1.5.1, I get this crash when starting my Meteor project:

```
TypeError: Cannot read property 'forEach' of undefined
     at checkNpmVersions (packages/tmeasday:check-npm-versions/check-npm-versions.js:30:3)
     at meteorInstall.node_modules.meteor.react-meteor-data.react-meteor-data.jsx (packages/react-meteor-data/react-meteor-data.jsx:3:1)
     at fileEvaluate (packages/modules-runtime.js:333:9)
     at require (packages/modules-runtime.js:228:16)
```

Af far as I can tell, the problem is caused by [this line](https://github.com/tmeasday/check-npm-versions/blob/f50c9860d820b856c353e93590a3cf33339ea2b8/check-npm-versions.js#L30).

In [this line](https://github.com/tmeasday/check-npm-versions/blob/f50c9860d820b856c353e93590a3cf33339ea2b8/check-npm-versions.js#L2), the code is simply importing `_` from `meteor/underscore`, but apparently, this no longer works.

As a quick experiment, I tried adding the underscore package manually (`meteor add underscore`), but nothing changes...

**Comment by csillag** [comment]
Later this problem disappeared without any further action on my part, so it must have been some intermittent issue. Closing this.

**Comment by fatbird** [comment]
I'm stuck on this write now, having updated from 1.4.3 to 1.5.2.  Any idea what fixed it?

**Comment by LaughingBubba** [comment]
Same

**Comment by fatbird** [comment]
Bit of shot in the dark, but I verified this was still a problem with 1.5.2.2, then removed the react-meteor-data package (taking this one out too), and then added it again, which brought it back at 0.2.15.  Don't have the problem anymore.

**Comment by matheus23** [comment]
For anyone else stumbling on this issue, this fixed it for me (after updating to 1.6.1):
```
meteor remove react-meteor-data && meteor add react-meteor-data
```
